Application/Api/Document/Standard/Repository/Billing/Billing.php

```diff
@@ -94,59 +89,47 @@ private function buildDocument($pageList = array())
     }
 
     /**
-     * @param $Text
-     * @param $ItemName
-     * @param $Year
-     * @param $TotalPrice
-     * @param $DebtorSalutation
-     * @param $DebtorFirstName
-     * @param $DebtorLastName
-     * @param $CauserSalutation
-     * @param $CauserFirstName
-     * @param $CauserLastName
-     * @param $Birthday
-     * @param $From
-     * @param $To
-     * @param $Date
-     * @param $Location
-     * @param $CompanyName
-     * @param $CompanyExtendedName
-     * @param $CompanyAddress
+     * @param string $Text
+     * @param string $ItemName
+     * @param string $Year
+     * @param string $TotalPrice
+     * @param string $DebtorSalutation
+     * @param string $DebtorFirstName
+     * @param string $DebtorLastName
+     * @param string $CauserSalutation
+     * @param string $CauserFirstName
+     * @param string $CauserLastName
+     * @param string $Birthday
+     * @param string $From
+     * @param string $To
+     * @param string $Date
+     * @param string $Location
+     * @param string $CompanyName
+     * @param string $CompanyExtendedName
+     * @param string $CompanyAddress
+     * @param string $StudentIdentifier
      *
      * @return string
      */
-    private function setPlaceholders(
-        $Text,
-        $ItemName,
-        $Year,
-        $TotalPrice,
-        $DebtorSalutation,
-        $DebtorFirstName,
-        $DebtorLastName,
-        $CauserSalutation,
-        $CauserFirstName,
-        $CauserLastName,
-        $Birthday,
-        $From,
-        $To,
-        $Date,
-        $Location,
-        $CompanyName,
-        $CompanyExtendedName,
-        $CompanyAddress
-    ) {
+    private function setPlaceholders($Text, $ItemName, $Year, $TotalPrice, $DebtorSalutation, $DebtorFirstName,
```

Comment:
Codacy found an issue: [The method setPlaceholders has 19 parameters. Consider reducing the number of parameters to less than 10.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)